docs/features/publish-to-github.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Publish to GitHub"
+description: "Learn more about the Publish to GitHub feature"
+---
+
+This integration allows users to publish their projects from the CodeCrafters platform to GitHub with just a couple of clicks.
+
+Each course available on CodeCrafters includes an option in the language dropdown menu to **"Publish to GitHub"**:
```

Comment:
Challenge, not course

Reply:
Also might be confusing that we're saying "each challenge" instead of "each repository".